src/ImageSharp/Formats/Webp/Lossy/YuvConversion.cs

```diff
@@ -312,6 +586,178 @@ public static void YuvToBgr(int y, int u, int v, Span<byte> bgr)
             bgr[0] = (byte)YuvToB(y, u);
         }
 
+#if SUPPORTS_RUNTIME_INTRINSICS
+
+        private static void ConvertYuvToBgrSse41(Span<byte> topY, Span<byte> bottomY, Span<byte> topDst, Span<byte> bottomDst, Span<byte> ru, Span<byte> rv, int curX, int step)
```

Comment:
These spans can be also converted to `ref`-s.

Reply:
I feel a bit more comfortable with using spans here, not sure if its really worth changing to ref's

Reply:
The main overhead seems to appear here:

https://github.com/SixLabors/ImageSharp/blob/984a72571db182b8a7c0eca83f9e9de6a9b162df/src/ImageSharp/Formats/Webp/Lossy/YuvConversion.cs#L620-L622

You can consider "de-spanning" only in this method and `ConvertYuv444ToRgbSse41` replacing `.Slice` with `Unsafe.Add` to y/u/v base references.

Reply:
ok i am fine with that: 6293f72c